monkey/monkey_island/cc/setup/agent_event_handlers.py

```diff
@@ -22,3 +29,16 @@ def _subscribe_and_store_to_event_repository(container: DIContainer):
         container.resolve(ICredentialsRepository)
     )
     agent_event_queue.subscribe_type(CredentialsStolenEvent, save_stolen_credentials_subscriber)
+
+
+def _subscribe_ping_scan_event(container: DIContainer):
```

Comment:
Where is this being called?

Reply:
+1 Travis complains about not being used anywhere.

Reply:
That's what you get when you only write unit tests :D